projects/smart-table/src/lib/smart-table/smart-table.component.html

```diff
@@ -15,26 +15,27 @@
         <div class="a-input__wrapper">
           <div auiFlyout size="small" align="right">
             <button class="a-button-outline has-icon"
-                    [attr.aria-label]="options.translations?.openColumnSelector || 'Toon/verberg kolommen'"
-                    [title]="options.translations?.openColumnSelector || 'Toon/verberg kolommen'" auiFlyoutAction>
+                    [attr.aria-label]="(configuration$ | async)?.options.translations?.openColumnSelector || 'Toon/verberg kolommen'"
+                    [title]="(configuration$ | async)?.options.translations?.openColumnSelector || 'Toon/verberg kolommen'"
+                    auiFlyoutAction>
               <span class="fa fa-columns"></span>
             </button>
 
             <div auiFlyoutZone>
               <div class="u-margin-xs">
                 <ul class="a-list a-list--reset aui-smart-table__column-checklist u-margin-bottom-xs">
                   <li class="a-list__item"
-                      *ngFor="let column of selectableColumns; index as i">
+                      *ngFor="let column of (selectableColumns$ | async); index as i">
                     <div class="a-input__checkbox">
-                      <input type="checkbox" id="{{ 'checkbox--' + i + '--' + column.id }}"
-                             name="{{ 'checkbox--' + i + '--' + column.id }}" [checked]="!column.hidden"
+                      <input type="checkbox" id="{{ 'checkbox--' + i + '--' + column.value }}"
```

Comment:
The type of the column is a TableColumn, which doesn't have an id, but rather a value. This was initially wrong in the code but didn't cause any problems since we still used the index of a for loop to create unique values.